less/components/agenda.less

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright (c) 2021 Eclipse Foundation, Inc.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * Contributors:
+ *   Martin Lowe <martin.lowe@eclipse-foundation.org>
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+*/
+
+section#plan-of-the-day .backdrop {
+    background-image: url('/images/first-banner.png');
+    background-size: cover;
+    background-position: bottom;
+  }
+  .eclipsefdn-agenda {
+    h2 {
+      color: @brand-primary;
+    }
+    .eclipsefdn-agenda-legend h4, .eclipsefdn-agenda-legend li {
+      color: @brand-secondary;
+    }
+    .eclipsefdn-agenda-legend li {
```

Comment:
Would it be better to make this li nested in .eclipsefdn-agenda-legend, like the h4 below?